lib/measures/tbd/resources/utils.rb

```diff
@@ -1449,6 +1449,9 @@ def overlaps?(p1 = nil, p2 = nil, id1 = "", id2 = "")
     area = area.get
     return false                                                 if area < TOL
 
+    delta = (area - area1 - area2).abs
+    return false                                                 if delta < TOL
```

Comment:
Fix to issue described [here](https://github.com/rd2/osut/pull/11#discussion_r1192481161), a potential issue for cases with frame&divider-enabled fenestration.